src/sharedeventfilter.cpp

```diff
@@ -42,7 +42,7 @@ bool SharedEventFilter::handleKeyEvent(QKeyEvent *e)
     bool ctrlPressed = e->modifiers() & Qt::ControlModifier;
     switch (e->key())
     {
-    case Qt::Key_F1:            QProcess::startDetached("dman dde");            return true;
+    case Qt::Key_F1:            QProcess::startDetached("dman", { "dde" });            return true;
```

Comment:
```suggestion
    case Qt::Key_F1:            QProcess::startDetached("dman", { "dde" });     return true;
```

（它之前的 return 那部分是对齐的

Reply:
好了